### PR TITLE
#305 added npm-audit feature in `SHOULDI`

### PR DESCRIPTION
**Pull request by Yash-Varshney**

### CODE REVIEW

File: examples/shouldi/.gitignore [hunk 1]
```diff
@@ -19,5 +19,6 @@ wheelhouse/
 *.db
 htmlcov/
 tests/golangci-lint-download/
+tests/javascript_algo-download/
```
Comment (pdxjohnny) [inline examples/shouldi/.gitignore]: Don't forget to add 

```
tests/npm-audit-download/
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `subflow` in `OperationImplementationContext`
 - Documentation on writing examples and running doctests
 - Doctestable Examples to high-level API.
+- Shouldi got an operation to run npm-audit on JavaScript code
```
Comment (pdxjohnny) [inline CHANGELOG.md]: Perfect! :)

File: examples/shouldi/.gitignore (hunk 1 above)
Comment (Yash-Varshney) [reply examples/shouldi/.gitignore]: oops ... missed it . Thanks @pdxjohnny .. Did it :)

File: CHANGELOG.md (hunk 2 above)
Comment (Yash-Varshney) [reply CHANGELOG.md]: Thanks 👍

**Comment by Yash-Varshney** [comment]
Done !! Thanks @pdxjohnny.
Won't be possible without your help and guidance !! 👍

**Comment by pdxjohnny** [comment]
Great job @Yash-Varshney!! Looking great. Merged :)

**Pull request merged**